CMakeLists.txt

```diff
@@ -882,13 +914,61 @@ else()
                       INCLUDES
                       ${draco_include_paths}
                       OBJLIB_DEPS
-                      draco_unity_plugin
+                      ${unity_objlib_dependency}
+                      LIB_DEPS
+                      ${unity_lib_dependency}
+                      )
+
+    draco_add_library(NAME
+                      dracoenc_unity
+                      TYPE
+                      ${unity_decoder_lib_type}
+                      DEFINES
+                      ${draco_defines}
+                      INCLUDES
+                      ${draco_include_paths}
+                      OBJLIB_DEPS
+                      draco_unity_enc_plugin
                       LIB_DEPS
                       ${draco_plugin_dependency})
 
     # For Mac, we need to build a .bundle for the unity plugin.
     if(APPLE)
       set_target_properties(dracodec_unity PROPERTIES BUNDLE true)
+      set_target_properties(dracoenc_unity PROPERTIES BUNDLE true)
+    endif()
+    if(IOS)
```

Comment:
It would be better update the flags for IOS in cmake/draco_build_definitions.cmake. Just add something like this to the bottom of draco_set_build_definitions().

```
if(IOS)
  list(APPEND draco_base_cxx_flags "-fembed-bitcode")
endif()
```